src/main/kotlin/com/sourcegraph/cody/listeners/CodyDocumentListener.kt

```diff
@@ -25,6 +25,7 @@ class CodyDocumentListener(val project: Project) : BulkAwareDocumentListener {
     }
   }
 
+  // todo: documentChanged
```

Comment:
Remove or make it more meaningful please.

Reply:
fixed